desktop/package/windows/Bisq.iss

```diff
@@ -64,6 +65,96 @@ begin
   Result := False;
 end;
 
+procedure DirectoryCopy(SourcePath, DestPath: string);
+var
+  FindRec: TFindRec;
+  SourceFilePath: string;
+  DestFilePath: string;
+begin
+  if FindFirst(SourcePath + '\*', FindRec) then
+  begin
+    try
+      repeat
+        if (FindRec.Name <> '.') and (FindRec.Name <> '..') then
+        begin
+          SourceFilePath := SourcePath + '\' + FindRec.Name;
+          DestFilePath := DestPath + '\' + FindRec.Name;
+          if FindRec.Attributes and FILE_ATTRIBUTE_DIRECTORY = 0 then
+          begin
+            if FileCopy(SourceFilePath, DestFilePath, False) then
+            begin
+              Log(Format('Copied %s to %s', [SourceFilePath, DestFilePath]));
+            end
+              else
+            begin
+              Log(Format('Failed to copy %s to %s', [SourceFilePath, DestFilePath]));
+            end;
+          end
+            else
+          begin
+            if DirExists(DestFilePath) or CreateDir(DestFilePath) then
+            begin
+              Log(Format('Created %s', [DestFilePath]));
+              DirectoryCopy(SourceFilePath, DestFilePath);
+            end
+              else
+            begin
+              Log(Format('Failed to create %s', [DestFilePath]));
+            end;
+          end;
+        end;
+      until not FindNext(FindRec);
+    finally
+      FindClose(FindRec);
+    end;
+  end
+    else
+  begin
+    Log(Format('Failed to list %s', [SourcePath]));
+  end;
+end;
+
+//Delete old app directory to prevent issues during update
+procedure DeleteOldAppDataDirectory;
+var
+  entry: String;
+begin
+  entry := ExpandConstant('{localappdata}') + '\Bisq\';
```

Comment:
Is {localappdata} correct? I don't see it purging the app folder anymore with the latest exe, like it did in the previous exe. Did you use {app} previously?

Reply:
hmm - worked for me. I didn't change anything there. I put a test file in the app folder in it was gone afterwards. I can have a look again tomorrow. How did you test it?

Reply:
My apologies, it is deleting it. I confirmed using process monitor.
Previously there was a noticeable delay between the purge (when done in InitializeSetup) and install, that I could see the folder being removed. But now that the purge is done in PrepareToInstall, it is not noticeable.